dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/FeatureRepository.java

```diff
@@ -371,6 +384,11 @@ static void writeFeatureAttributes(ImmutableAttributes iAttr, ProvisioningDetail
 
         out.writeUTF(iAttr.activationType.toString());
 
+        out.writeShort(iAttr.alternateNames.size());
```

Comment:
Other places that write a collection use writeInt for the size.  I agree that it is far fetched to ever need that big of a size for alternate names, but I'd error on the side of consistency here.